src/mumble/ClientUser.cpp

```diff
@@ -207,6 +208,11 @@ void ClientUser::setLocalMute(bool mute) {
 	emit muteDeafStateChanged();
 }
 
+void ClientUser::setLocalVolume(float volume) {
+	fLocalVolume = volume;
+	emit muteDeafStateChanged();
```

Comment:
Why do you need this?

If this is not needed, I think you should just simply assign to fLocalVolume like I do in my change.